manager/controlapi/node.go

```diff
@@ -283,7 +283,7 @@ func (s *Server) RemoveNode(ctx context.Context, request *api.RemoveNodeRequest)
 				return grpc.Errorf(codes.FailedPrecondition, "node %s is a cluster manager and is a member of the raft cluster. It must be demoted to worker before removal", request.NodeID)
 			}
 		}
-		if node.Status.State == api.NodeStatus_READY {
+		if !request.Force && node.Status.State == api.NodeStatus_READY {
```

Comment:
I think it's okay to only allow force-removing workers. If you want to force-remove a manager, you can demote it first and then force-remove it. Maybe in the future we can make `node rm --force` do the demotion automatically, but I don't think it's as important as the basic feature in this PR.